sources/Renderer/Vulkan/VKRenderSystem.cpp

```diff
@@ -821,16 +821,21 @@ void VKRenderSystem::CreateInstance(const RendererConfigurationVulkan* config)
         );
     };
 
+    /* Setup Vulkan instance descriptor */
+    VkInstanceCreateInfo instanceInfo = {};
+
     for (const VkExtensionProperties& prop : extensionProperties)
     {
         const VKExtSupport extSupport = GetVulkanInstanceExtensionSupport(prop.extensionName);
         if (IsVKExtSupportIncluded(extSupport))
             extensionNames.push_back(prop.extensionName);
+#ifdef VK_KHR_PORTABILITY_ENUMERATION_EXTENSION_NAME
+        if (strcmp(prop.extensionName, VK_KHR_PORTABILITY_ENUMERATION_EXTENSION_NAME) == 0)
+            instanceInfo.flags |= VK_INSTANCE_CREATE_ENUMERATE_PORTABILITY_BIT_KHR;
+            extensionNames.push_back(prop.extensionName);
```

Comment:
If adding the extension name is meant to be part of the `if`-statement, then that statement must have a compound statement, i.e. it's missing the curly braces `{`/`}`.
Besides that, can you please replace the statement that adds the extension name by adding it to [`IsVulkanInstanceExtOptional()`](https://github.com/LukasBanana/LLGL/blob/master/sources/Renderer/Vulkan/Ext/VKExtensionRegistry.cpp#L78C13-L78C40) instead. This will add the name in the conditional block above. We can keep the rest as a special case to add this bit field.

Reply:
(oh forget to add {}, I prefer always put {} but like it look like you prefer remove them for line I iniatially don't add them but forgot to put it when I add the second line)